ons_alpha/bulletins/models.py

```diff
@@ -121,6 +122,19 @@ def get_context(self, request, *args, **kwargs):
         context["toc"] = self.toc
         return context
 
+    def save(self, *args, **kwargs):
+        if self.revisions.exists():
+            latest_revision = self.revisions.order_by("-created_at").first()
+            if latest_revision:
+                latest_correction_block = None
+                for block in self.updates:  # pylint: disable=not-an-iterable
+                    if block.block_type == "corrections_notices_story":
+                        latest_correction_block = block
+                        break
+                if latest_correction_block and "previous_version" not in latest_correction_block.value:
```

Comment:
Issue: It's not about the field not being there, it's about it being blank. You'll want to add a block to the struct that's already there, but just hide it from the admin (see notes on ticket).